### ISSUE TITLE
Icons and visiblity

### ISSUE THREAD

**Issue by wpsimon09** [FELIXroboticsBV/FELIX-FILO#21]
Icons in the application are not visible in the light mode because they were designed for the dark mode. For this reason it is necessary to use pngs instead of the svgs which can be achieved by only setting hard coded path to the icons.
